src/components/font_size_editor/font_size_editor.ts

```diff
@@ -31,6 +31,14 @@ css/* scss */ `
     input::-webkit-inner-spin-button {
       -webkit-appearance: none;
     }
+
```

Comment:
menu-item horizontal padding went down 16px -> 13px

Reply:
Necessary so as not to have too much padding before the icons,